code/game/machinery/autolathe/artist_bench.dm

```diff
@@ -0,0 +1,405 @@
+#define ERR_OK 0
+#define ERR_NOTFOUND "not found"
+#define ERR_NOMATERIAL "no material"
+#define ERR_NOREAGENT "no reagent"
+#define ERR_NOLICENSE "no license"
+#define ERR_PAUSED "paused"
+#define ERR_NOINSIGHT "no insight"
+#define MAX_STAT_VALUE 12
+
+/obj/machinery/autolathe/artist_bench
+	name = "artist's bench"
+	desc = "Insert wood, steel, glass, plasteel, plastic and a bit of your soul to create a beautiful work of art. \
+			Requires a stock of at least 5 sheets of each of the aforementioned materials before you begin sacrificing your soul."	// OCCULUS EDIT - Makes its requirements clear
```

Comment:
Why are we reducing this from the Eris requirements? Steel, glass, and plastic are super plentiful.

Reply:
It's not reduced, that is simply the minimum stock of materials required. What it consumes is independent of that.